Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -8549,6 +8549,18 @@ - (void)actionSheetAssistant:(WKActionSheetAssistant *)assistant getAlternateURL
         completion(nil, nil);
 }
 
+- (NSArray<UIMenuElement *> *)additionalMediaControlsContextMenuItemsForActionSheetAssistant:(WKActionSheetAssistant *)assistant
+{
+#if PLATFORM(VISION)
+    if (self.webView.fullscreenState != WKFullscreenStateInFullscreen)
+        return @[];
+
+    return @[ [self.webView fullScreenWindowSceneDimmingAction] ];
+#else
+    return @[];
+#endif
```

Comment:
Extreme nit - you might be able to make this a _tiny_ bit more succinct:

```suggestion
#if PLATFORM(VISION)
    if (self.webView.fullscreenState == WKFullscreenStateInFullscreen)
        return @[ [self.webView fullScreenWindowSceneDimmingAction] ];
#endif
    return @[ ];
```

Reply:
Done!